src/main/java/DateTime.java

```diff
@@ -0,0 +1,22 @@
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+public class DateTime{
+    protected String input;
+    protected Date date;
+
+    public DateTime(String input) throws ParseException {
+        String dateFormat = "dd-MM-yyyy HH:mm";
+        SimpleDateFormat formatter = new SimpleDateFormat(dateFormat);
+        this.input = input;
+        this.date = formatter.parse(input);
+    }
+
+    @Override
+    public String toString() {
+        String dateFormat = "dd-MM-yyyy HH:mm";
+        SimpleDateFormat formatter = new SimpleDateFormat(dateFormat);
+        return formatter.format(this.date);
+    }
+}
```

Comment:
It might look better in GitHub if you can add newline at the end of every file, because otherwise there will be a red symbol that appears indicating that there is no newline.